docs/source/user_guide/gui.md

```diff
@@ -0,0 +1,174 @@
+(target-gui)=
+# Graphical User Interface
+
+The `movement` graphical user interface (GUI), powered by our custom plugin for
+[napari](napari:), makes it easy to view and explore `movement`
+motion tracks. Currently, you can use it to
+visualise 2D [poses datasets](target-poses-and-bboxes-dataset)
+as points overlaid on video frames.
+
+:::{warning}
+The GUI is still in early stages of development but we are working on ironing
+out the [kinks](https://github.com/neuroinformatics-unit/movement/issues?q=sort%3Aupdated-desc+is%3Aissue+state%3Aopen+label%3AGUI+label%3Abug).
```

Comment:
```suggestion
out the [kinks](movement-github:issues?q=sort%3Aupdated-desc+is%3Aissue+state%3Aopen+label%3AGUI+label%3Abug).
```
untested

Reply:
Thanks for the catch, tested it locally, and it works. I'll accept the suggestion as is.